typespec-tests/src/main/java/com/type/model/inheritance/nesteddiscriminator/models/Salmon.java

```diff
@@ -116,11 +116,11 @@ public Salmon setPartner(Fish partner) {
     @Override
     public JsonWriter toJson(JsonWriter jsonWriter) throws IOException {
         jsonWriter.writeStartObject();
-        jsonWriter.writeStringField("kind", "salmon");
         jsonWriter.writeIntField("age", getAge());
+        jsonWriter.writeStringField("kind", "salmon");
```

Comment:
This is directly generated, or manual change?

Reply:
This is directly generated, so removed now.